tests/main/lxd-mount-units/task.yaml

```diff
@@ -34,6 +37,7 @@ execute: |
     if os.query is-ubuntu 22.10; then
         CODENAME=$(. /etc/os-release && echo "$VERSION_CODENAME")
         lxc launch --quiet "images:ubuntu/$CODENAME" ubuntu
+        core_snap=core22
```

Comment:
We don't need this, do we? Since you already assign this before.

Reply:
we assign core20 as core snap, but since 22.10 the core snap used is core22. I'll add a change in os.query to allow to ask something like "os.query after-ubuntu 22.10"